lib/filterrific/version.rb

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
 
 module Filterrific
-  VERSION = "5.2.2"
+  VERSION = "5.2.2.2"
```

Comment:
Let's leave the version as is. I will update it when I release a new version. Unless you need to change the version?